### ISSUE TITLE
Not able to set it up locally.

### ISSUE THREAD

**Issue by dileephell** [awslabs/aws-lex-web-ui#17]
I am trying to run the sample AWS-Lex-Web-UI from the https://github.com/awslabs/aws-lex-web-ui#sample-site. As per the directions i am able to create the cognito pool id and also saved in the chatbot-ui-loader-config.json and tried with npm start. Server started at localhost:8000 but i am not able to run any one of the bot command.

Does anybody already implemented in the WEB-UI part using the sample example. I want to export(newly created one) my bot from AWS-LEX to my local server. So that i can run it locally.

![err](https://user-images.githubusercontent.com/2589018/34107451-432c81ba-e422-11e7-85a3-b944deafc46d.png)

**Comment by atoa** [comment]
I wonder if there's something missing with the config. Please note that in addition to the Cognito Pool Id, you'd need to configure the bot name.

Do you see any errors in the browser console?
Any chance you can post the contents of your `chatbot-ui-loader-config.json` here?

-- Oliver

**Comment by jhnferraris** [comment]
@dileephell Also checkout the browser console for any errors. It helped me out trying this one out.
Here are some of the things that I experienced before I was able to try this out locally:

- IAM permissions should be properly set such as cognito pools should have access to the Lex or that Polly should be able to access Lex.
- Federated identities versus User Pools - I had to use Federated Identity pool

Hope this helps.
